src/components/InputNewTodo/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styles from './InputNewTodo.module.css'
+
+type InputNewTodoProps = {
+    todoTitle: string,
+    onChange: (todoTitle: string) => void,
+    onSubmit: (todo: any) => void,
```

Comment:
provide a correct type instead of `any` use `Todo` (defined inside `src/MainApp/index.tsx`)